client/entrypoints/payment-gateways/style.scss

```diff
@@ -7,27 +7,10 @@
 				margin-bottom: 12px;
 			}
 		}
-	}
-}
 
-.wcstripe-confirmation-modal {
```

Comment:
This is no longer necessary, using the styles in `client/components/confirmation-modal/alert-title.js`